platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -8,172 +8,116 @@ tags: [azure, batch, compute environment]
 :::note
 This guide assumes you already have an Azure account with a valid Azure Subscription.
 For details, visit [Azure Free Account][az-create-account].
-Ensure you have sufficient permissions to create resource groups, an Azure Storage account, and an Azure Batch account.
+Ensure you have sufficient permissions to create resource groups, an Azure Storage account, and a Batch account.
```

Comment:
```suggestion
Ensure you have sufficient permissions to create resource groups, an Azure Storage account, and an Azure Batch account.
```
I would keep this as Azure Batch rather than Batch. It's the first use on this page, and IMO, it isn't as clear. Ignore if you disagree.